src/Processor/UpdatedOrderProcessorInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusOrderEditPlugin\Processor;
+
+use Sylius\Component\Core\Model\OrderInterface;
+
+interface UpdatedOrderProcessorInterface
+{
+    public function process(OrderInterface $newOrder): OrderInterface;
```

Comment:
```suggestion
    public function process(OrderInterface $order): OrderInterface;
```

or

```suggestion
    public function process(OrderInterface $updatedOrder): OrderInterface;
```

to be consistent with the service name.